src/udiskslinuxdrive.c

```diff
@@ -50,6 +51,71 @@
 #include "udisksdaemonutil.h"
 #include "udiskslinuxdevice.h"
 
+static const gchar *
+_get_blk_path (UDisksDrive *drv, GDBusMethodInvocation *invocation);
```

Comment:
Any reason for that? There are four more static functions below that don't start with underscore. ``static`` is what makes things "internal" to the object (file) so does omitting these from the header files. Do we really need a third way to mark things as not public?